index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="UTF-8">
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav class="menu">
+                <a href="#" class="logo"><h2>My Logo</h2></a>
```

Comment:
Good thing you made your logo an `a` tag. But now you don't need the `h2` tag anymore.

- [ ] Kindly, remove the `h2` tag, because, as the name says, the `hx` tags are headers. No need to replace it with something else, but is enough if you remove it.

- [ ] As explained by the previous reviewer, the logo needs to be outside the `nav` tag but inside the `header`. Please ensure this happens.